src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -11,7 +19,504 @@ internal class CodeStylePage : AbstractOptionPage
     {
         protected override AbstractOptionPageControl CreateOptionPage(IServiceProvider serviceProvider)
         {
-            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s));
+            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s), GetCurrentEditorConfigOptionsString, LanguageNames.CSharp);
+        }
+
+        internal static string GetCurrentEditorConfigOptionsString(OptionSet optionSet)
+        {
+            var editorconfig = new StringBuilder();
+            // Core EditorConfig Options
+            editorconfig.AppendLine("###############################");
+            editorconfig.AppendLine("# Core EditorConfig Options   #");
+            editorconfig.AppendLine("###############################");
+            editorconfig.AppendLine("# You can uncomment the next line if this is your top-most .editorconfig file.");
+            editorconfig.AppendLine("# root = true");
```

Comment:
I think vast majority of users would have a single .editorconfig so I'd prefer root = true to be not commented out by default.